src/VirtoCommerce.Platform.Data/ChangeLog/ChangeLogService.cs

```diff
@@ -98,9 +76,41 @@ public virtual async Task DeleteAsync(string[] ids)
                     repository.Remove(entity);
                 }
                 await repository.UnitOfWork.CommitAsync();
-                ChangeLogCacheRegion.ExpireRegion();
+                Reset();
             }
         }
+        #endregion
+
+        #region ILastModifiedDateTime Members
+        public DateTimeOffset GetLastModified(string entityName = null)
+        {
+            DateTimeOffset result;
+
+            entityName ??= string.Empty;
+
+            var cacheKey = CacheKey.With(GetType(), "LastModifiedDateTime", entityName);
+            result = _memoryCache.GetOrCreateExclusive(cacheKey, options =>
+            {
+                options.AddExpirationToken(ChangeLogCacheRegion.CreateChangeTokenForKey(entityName));
+
+                return DateTimeOffset.UtcNow;
+            });
```

Comment:
If requested entity name is not used by platform, its modification date will be cached and then will never be updated. For example, if we override some entity of VC modules, storefront will also need to be updated to use the type name of new entity - otherwise, its cache will not work properly. It could be safer to always return `DateTime.UtcNow` for such cases - however, I can't think of any way to detect that.